src/AutoSplitImage.py

```diff
@@ -140,6 +169,10 @@ def compare_with_capture(
         default: "AutoSplit | int",
         capture: MatLike | None,
     ):
+        """Extract image text from rectangle position and compare it with the expected string."""
+        if self.text is not None:
+            return extract_and_compare_text(capture[self.__y:self.__yy, self.__x:self.__xx], self.text)
+
         """Compare image with capture using image's comparison method. Falls back to combobox."""
```

Comment:
This breaks the docstring for `compare_with_capture`. Something like this would be better:

```suggestion
        """
        Compare image with capture using image's comparison method. Falls back to combobox.
        For OCR text files: extract image text from rectangle position and compare it with the expected string.
        """
        if self.text is not None:
            return extract_and_compare_text(capture[self.__y:self.__yy, self.__x:self.__xx], self.text)
```

Reply:
Done